launch_ros/launch_ros/substitutions/find_package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the FindPackage substitution."""
+
+from typing import List
+from typing import Text
+
+from ament_index_python.packages import get_package_prefix
+
+from launch.launch_context import LaunchContext
+from launch.some_substitutions_type import SomeSubstitutionsType
+from launch.substitution import Substitution
+from launch.utilities import normalize_to_list_of_substitutions
+from launch.utilities import perform_substitutions
+
+
+class FindPackage(Substitution):
+    """
+    Substitution that tries to locate the package prefix of a ROS package.
+
+    The ROS package is located using ament_index_python.
+
+    :raise: ament_index_python.packages.PackageNotFoundError when package is
+        not found during substitution
```

Comment:
@ivanpauno consider:
```suggestion
        not found during substitution.
```